### PR TITLE
DOCS-1237: Removing  junk characters from the yaml file

### PR DESCRIPTION
**Pull request by girishshankaran**
https://tigera.atlassian.net/browse/DOCS-1237

### CODE REVIEW

**Comment by ctauchen** [comment]
@girishshankaran The changes you have above are good, but they also apply to the analogous includes for ce 3.15 and ce 3.14.

Additionally, I see this pattern in another location: calico/operations/image-options/alternate-registry.mdx (next, 3.25, 3.24)

The fix is good -- I like that line highlighting.

@ctauchen Yes, my bad. I should have fixed in other places as well.

File: calico-enterprise_versioned_docs/version-3.14/_includes/components/PrivateRegistryRegular.js [hunk 1]
```diff
@@ -137,7 +137,8 @@ spec:
   variant: TigeraSecureEnterprise
   imagePullSecrets:
     - name: tigera-pull-secret
-  <b>registry: myregistry.com</b>`}</CodeBlock>
+    // highlight-next-line
+      registry: myregistry.com<`}</CodeBlock>
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.14/_includes/components/PrivateRegistryRegular.js]:
```suggestion
      registry: myregistry.com`}</CodeBlock>
```

File: calico_versioned_docs/version-3.25/operations/image-options/alternate-registry.mdx [hunk 2]
```diff
@@ -81,5 +81,6 @@ spec:
   variant: Calico
   imagePullSecrets:
     - name: tigera-pull-secret
-  <b>registry: myregistry.com</b>
+    // highlight-next-line
+      registry: myregistry.com<
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.25/operations/image-options/alternate-registry.mdx]:
```suggestion
      registry: myregistry.com
```

**Review by ctauchen — DISMISSED** [review]
LGTM pending two small typo fixes.

File: calico-enterprise_versioned_docs/version-3.14/_includes/components/PrivateRegistryRegular.js (hunk 1 above)
Comment (girishshankaran) [reply calico-enterprise_versioned_docs/version-3.14/_includes/components/PrivateRegistryRegular.js]: @ctauchen it is done

File: calico_versioned_docs/version-3.25/operations/image-options/alternate-registry.mdx (hunk 2 above)
Comment (girishshankaran) [reply calico_versioned_docs/version-3.25/operations/image-options/alternate-registry.mdx]: @ctauchen it's done

**Review by ctauchen — APPROVED** [review]

**Pull request merged**